com.adsbynimbus.nimbus/Editor/AndroidPostBuildProcessor.cs

```diff
@@ -116,11 +116,15 @@ public void OnPostGenerateGradleAndroidProject(string path)
 				#endif
 				#if NIMBUS_ENABLE_ADMOB
 					builder.AppendLine(AndroidBuildDependencies.AdMobNimbusBuildDependency());
-					builder.AppendLine(AndroidBuildDependencies.AdMobCollectionFixBuildDependency());
+					builder.AppendLine(AndroidBuildDependencies.CollectionFixBuildDependency());
 				#endif
 				#if NIMBUS_ENABLE_MINTEGRAL
 					builder.AppendLine(AndroidBuildDependencies.MintegralBuildDependency());
 				#endif
+				#if NIMBUS_ENABLE_UNITY_ADS
+					builder.AppendLine(AndroidBuildDependencies.UnityAdsBuildDependency());
+					builder.AppendLine(AndroidBuildDependencies.CollectionFixBuildDependency());
```

Comment:
This is going to write multiple instances of `androidx.collection:collection:1.4.5` in the build file output if AdMob and Unity are both enabled. 

Is it possible to ensure this is only written once?